package.json

```
@@ -35,10 +35,10 @@
         "not dead"
     ],
     "scripts": {
-        "postinstall": "lerna bootstrap --no-ci",
-        "start": "lerna run start --parallel --scope=\"react-vapor\" --scope=\"coveo-styleguide\"",
+        "postinstall": "lerna bootstrap --no-ci --hoist",
```

I think we should use a `setup` task the same way we do it in the admin, otherwise we might have the same problem we had with Windows

```suggestion
        "setup": "lerna bootstrap --no-ci --hoist",
```

---

Since we already had more than one package and it was working that way for windows users, I will leave it be like this for now, but I'll keep it in mind.